app/components/shared/dashboard_component/dashboard_component.html.slim

```diff
@@ -0,0 +1,16 @@
+.dashboard__header
+  .govuk-width-container
+    h1.govuk-heading-m class="govuk-!-margin-bottom-2"
+      = @heading
+
+  .govuk-width-container
+    - if link.present?
+      .filters-information.moj-action-bar
+        = govuk_button_to @link[:text], @link[:url], method: :post, class: "govuk-!-margin-bottom-0 govuk-!-margin-top-0 dashboard__link"
```

Comment:
When we do `attr_accessor` or `attr_reader` you can just access the attribute by name, you don't need to access the instance variable. (Same applies to line 4 and line 14)

Reply:
Also, `method` is `:post` by default, so no need to specify that for `govuk_button_to`